pkg/execution/queue/item.go

```diff
@@ -222,6 +226,16 @@ type Throttle struct {
 	KeyExpressionHash string `json:"-"`
 }
 
+type Singleton struct {
+	// Key is the unique singleton key that's used to group queue items when
+	// processing singleton items.
+	Key string `json:"k"`
+
+	// Mode defines the behavior when a new singleton run is queued while another is active.
+	// It determines whether to ignore the new run or cancel the current one and replace it.
+	//Mode enums.SingletonMode `json:"m"`
```

Comment:
you should still enable this, and just ignore and do nothing for cancel mode for now.